### DISCUSSION TITLE
reuse props from `org, rov`

### DISCUSSION THREAD

**Discussion by VladimirAlexiev** [eprocurementontology/eprocurementontology#545] [Ideas]
This is a sub-issue of https://github.com/OP-TED/ePO/issues/295, listing some specific props that can be reused.

ePO reuses org:Organization, but defines its own props for some that can be found in `org` and `rov` (aka `regorg`).
Since both of these are EU core vocabs and published by OPOCE, I think they should be reused to the max.

- `epo:registeredAddress`: consider `org:hasRegisteredSite` (there's also `org:hasPrimarySite`). `Site` is not quite the same as `Address` but I got advice from Maxx Dekkers that it's ok to collapse them (as we did in euBusinessGraph
- `epo:hasLegalFormType`: use `rov:orgType`, and use skos:Concept instead of Literal
- `epo:hasOrganisationUnit`: use `org:hasUnit` (with inverse `org:unitOf`)
- `epo:hasLegalName`: use rov:legalName. Also, "cardinality  1" is wrong because companies in some countries have several legal names in several official languages
- `epo:hasMainClassification`: is not attached to any class! Consider using `rov:orgActivity`
  -  "The activity of a company should be recorded using a controlled vocabulary expressed as a SKOS concept scheme. Several such vocabularies exist, many of which map to the UN's ISIC codes. Where a particular controlled vocabulary is in use within a given context, such as SIC codes in the UK, it is acceptable to use these, however, the preferred choice for European interoperability is NACE".
  -  In your case you use CPV for classification: I think that's ok with the intent and description of `rov:orgActivity`

----

This reuse can spare you some modeling errors (eg  #402) or give good ideas (eg #403)

**Comment by giorgialodi** [comment]
this is also related to #386
